PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeRequest.java

```diff
@@ -78,6 +79,15 @@ public void setShippingAddressEditable(boolean shippingAddressEditable) {
         this.shippingAddressEditable = shippingAddressEditable;
     }
 
+    /**
+     * Grabs the shipping address editable field
```

Comment:
I think we can use a similar docstring here to what we use in standard PayPal request: https://github.com/braintree/braintree_android/blob/b2ced4e6bc983a10a5ae7d8eebddcb8808f97c7d/PayPal/src/main/java/com/braintreepayments/api/PayPalRequest.java#L89-L96

Reply:
Ah, but that string is for the setter. Do we have a setter for this? In any case I think we can mimic the docstrings there for all of these properties (same with the ones below)

Reply:
Ah, no I was confused - I didn't expand things and see the setter above with the same docstring as the docs I linked. 🤦